source/bot.js

```diff
@@ -111,6 +112,20 @@ client.on(Events.InteractionCreate, interaction => {
 			const choices = unfilteredChoices.filter(choice => choice.value.toLowerCase().includes(focusedOption.value.toLowerCase()))
 				.slice(0, 25);
 			interaction.respond(choices);
+		} else if (interaction.isContextMenuCommand()) {
+			const contextMenu = getContextMenu(interaction.commandName);
+			if (contextMenu.premiumCommand && !premium.paid.includes(interaction.user.id) && !premium.gift.includes(interaction.user.id)) {
+				interaction.reply({ content: `The \`/${interaction.commandName}\` command is a premium command. Learn more with ${commandMention("premium")}.`, ephemeral: true });
+				return;
+			}
+
+			const cooldownTimestamp = contextMenu.getCooldownTimestamp(interaction.user.id, interactionCooldowns);
+			if (cooldownTimestamp) {
+				interaction.reply({ content: `Please wait, the \`/${interaction.commandName}\` command is on cooldown. It can be used again <t:${cooldownTimestamp}:R>.`, ephemeral: true });
```

Comment:
```suggestion
				interaction.reply({ content: `Please wait, the \`/${interaction.commandName}\` context menu option is on cooldown. It can be used again <t:${cooldownTimestamp}:R>.`, ephemeral: true });
```
Wording update for accuracy here too